src/ducks/files/Toolbar.jsx

```diff
@@ -29,6 +29,8 @@ class Toolbar extends React.Component {
 
   render () {
     const { t, disabled, displayedFolder, actions, onSelectItemsClick, uploadFiles } = this.props
+    const notRootfolder = displayedFolder && displayedFolder.path !== '/'
+    console.log(displayedFolder);
```

Comment:
yeah I know for the log, i'll remove it next commit.
Yeah I messed up, I compared the `ROOT_DIR_ID` with `displayedFolder.dir_id` and it wouldn't work (obviously), but it works way better when i compare with `displayedFolder.id`
Commit incoming